libsovtoken/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "libsovtoken"
 
-version = "0.9.5"
+version = "0.9.6"
```

Comment:
Here we should also change version of `indy` crate to be 1.6.8.
We should stay consistent in version although nothing have changed there.